plugins/services/src/js/components/SDKPlans.tsx

```diff
@@ -22,6 +22,7 @@ import {
 import gql from "graphql-tag";
 import Loader from "#SRC/js/components/Loader";
 import RequestErrorMsg from "#SRC/js/components/RequestErrorMsg";
+// @ts-ignore
```

Comment:
Let's add a `// TODO` here, and file a Jira issue for this?

Reply:
will propose to add an ongoing effort to reduce the amounts of `any`s, apparently bad typings and ignore-comments later on.